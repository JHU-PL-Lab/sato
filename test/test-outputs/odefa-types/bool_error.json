[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "a" ],
          "err_match_val": "0",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "b" ],
          "err_match_val": "1",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "c = a and b",
    "err_steps": 68
  }
]