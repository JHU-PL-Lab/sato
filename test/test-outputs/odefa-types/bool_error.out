Type errors for:
- Input sequence  : []
- Found at clause : c = a and b
--------------------
* Value    : a = 0
* Expected : bool
* Actual   : int
--------------------
* Value    : b = 1
* Expected : bool
* Actual   : int
2 errors found. No further control flows exist.
